projects/plugins/jetpack/extensions/blocks/one-payment/variations.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * The different payment blocks that can be chosen.
```

Comment:
This is going to need to confirm somehow that the blocks can actually be chosen,  i.e. they have the correct plan, the correct features are available, etc.

While this only includes two blocks, perhaps this could be done at a first pass by checking in PHP land that both of the potential inner blocks are registered before registering this one.